includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -123,7 +123,7 @@ public function register_routes() {
 						'type'        => 'string',
 					],
 					'account_business_url'              => [
-						'description' => __( 'The business’s publicly available website.', 'woocommerce-payments' ),
+						'description' => __( 'git pu', 'woocommerce-payments' ),
```

Comment:
A typo?

Reply:
Looks like I did this typo while rebasing, sorry! Fixed.